Add tutorial notebook on inspection of DIA sources

vpetrecca opened the pull request:
Hi!
I created this tutorial notebook showing how to get lightcurves of Supernovae, along with cutouts (coadd,template,calexp,diffexp). It uses materials from the other tutorials, but everything is stacked together in a single code. The final products are the same I produced for this exploration https://community.lsst.org/t/lightcurve-offset-on-difference-images/6912

MelissaGraham reviewed (changes requested):
Awesome! 

Just two minor changes to request:
 (1) Please remove the logo of Rubin Observatory (or replace it with a logo for your own institution).
 (2) Please change credit statement to: "Developed by Vincenzo Petrecca. Please consider acknowledging Vincenzo if this notebook is used for the preparation of journal articles, software releases, or other notebooks. This notebook is based in part on material originally developed by Leanne Guy, Melissa Graham, Jeff Carlin and the Rubin Community Engagement Team for Data Preview 0."

And then a few notes, just FYI (or perhaps you already know):
 - new truth tables are projected to be released in the next two weeks, and it will no longer be necessary to use the DP0.1 truth_match table
 - the ForcedSourceOnDiaObject catalog became available last week, that will be useful for light curves
 - a new tutorial on the image cutout service is in the works, and should avoid having to use the Butler for image access

vpetrecca commented:
Hi Melissa, thank you for the suggestions. 
- I added comments mentioning the new truth table and the new cutouts notebook being released soon. 
- I thought the ForcedSourceOnDiaObject catalog was not available yet, but now I have updated the notebook adding a few cells about it. 
- I also added how to plot error bars on the plots.

Please, let me know your thoughts on this new version.

P.S. This is my first experience with pull requests. Sorry if I am doing something wrong.

MelissaGraham commented:
You're doing it all right! This review stage is mostly a formality, to make sure contributors aren't, e.g., accidentally removing other people's files (more helpful for people who are new to git). It's not really a review of the content (I confirmed the NB executes all the way through, though) -- and thanks for making the change with the logo and the credits statement.

Review approved for merge.
